### PR TITLE
Feature/se 56

### LINKED ISSUE #56: Create the Front-end for Job profile Skill set
Create Job profile Job Profile

### PR DESCRIPTION
**Pull request by bishop6232**
closes SE2-WS23/documentation#56

Screenshot 2023-11-29 at 09 49 16

implemented skillSet Page

### CODE REVIEW

**Review by Arkcorner — COMMENTED** [review]
Looks Good waiting for another Persons approval and will then approve

File: src/pages/Skills/SkillSetPage.jsx
Comment (j0world) [inline src/pages/Skills/SkillSetPage.jsx]: Please change the "For" in your heading to lowercase according to title case

**Comment by j0world** [comment]
The Skill set list doesn't open at the top for me, which others don't do either so that should be fine.
Otherwise it would be kind of you to either add an empty handleSaveChangesButtonClick() or print the selected skills to show the next person where they need to work or which const saves the selected skills.
I don't know whether that is a necessity though.

**Review by Arkcorner — APPROVED** [review]
Looks up to standard

**Pull request merged**
